src/pages/en/guides/content-collections.mdx

```diff
@@ -274,28 +274,28 @@ const enDocs = await getCollection('docs', ({ id }) => {
 ---
 ```
 
-### `getEntry()`
+### `getEntryBySlug()`
 
-`getEntry()` is function that returns a specific entry in a collection by entry ID (file path relative to the collection). Both of these are required parameters.
+`getEntryBySlug()` is function that returns a specific entry in a collection by its slug (a slug that can be used in a URL). Both of these are required parameters.
```

Comment:
```suggestion
`getEntryBySlug()` is function that returns a specific entry in a collection by its slug (filename without the extension). Both of these are required parameters.
```
Is this what it is? Is the slug exactly the file name (minus extension), or is it user-defined?

Saying "*a* slug" makes it sound like it could be up for grabs. If  something defined in frontmatter, then maybe we could say something like:

> (a value defined in the frontmatter)

Otherwise, the rest of the changes look good to me!

Reply:
Yes, I think linking to the term below is good. Why try to figure out how to say it twice?  😄 

I guess the closest heading is `#data-returned-from-a-collection-query`

If you'd rather, what about something like:

> its slug (URL-ready)

> its URL-ready slug

Does that convey it well enough?

Reply:
Yeah, I think either of those work well enough.

Reply:
Went with `its slug (URL-ready)`